Added bmc creds upgrade note (#1850)
theforeman/theforeman.org: _includes/manuals/3.0/1.2_release_notes.md

* The `host_param_true?` and `host_param_false?` macros changed the meaning of the `default_value` parameter. The value is now used for the comparison if the parameter is not set on the host, instead of returing the value directly. Please update your templates accordingly.
* The Debian packages are signed with a new key ([5B7C3E5A735BCB4D615829DC0BDDA991FD7AAC8A](/static/keys/5B7C3E5A735BCB4D615829DC0BDDA991FD7AAC8A.pub)). Please import it to verify the packages.
* Foreman 2.5 for Debian/Ubuntu had briefly wrong versioned Ansible and Remote Execution plugins published, please read our [blog post how to properly upgrade these](/2021/08/please-manually-upgrade-ansible-and-remote-execution-plugins-on-25.html).

> * Setting named "BMC credentials access", which controls BMC credentails access through ENC YAML output and templates, is turned off by default for increased security. To turn it on, go to Administer - Settings.

Reviewer: Perhaps we can also mention the exact setting name which must be changed.

Author: What do you mean, I did mention exactly that:

```
$ ag "BMC credentials access"
app/models/setting/auth.rb
29:      set('bmc_credentials_accessible', N_("Permits access to BMC interface passwords through ENC YAML output and in templates"), true, N_('BMC credentials access')),
```

Do you mean to mention Authentication tab?

Reviewer: Ah right, you did. I didn't think about about the first sentence so I found it hard to connect. How about:
```suggestion
* BMC credentials access through ENC YAML output and templates is turned off by default for increased security. To turn it on, go to Administer - Settings and turn the "BMC credentials access" setting on.
```

Author: Sure, amended.
